README.md

```diff
@@ -1 +1,148 @@
-# math-magicians2
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>Math Magicians App</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math-Magician] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] @Daniel-Kigozi great work on your app, however, you still have section description, kindly remove them from all sections , also it would be nice if you added npm start as a way to run your project